### ISSUE TITLE
dig does not require to specify a resolver explicitly

### ISSUE THREAD

**Issue by MEschenbacher** [lixiangzhong/dnsutil#7]
With `dig`, one may omit the `@8.8.8.8` option to provide a resolver. This lib requires users to explicitly set a resolver.

Is there a way around this requirement?

**Comment by lixiangzhong** [comment]
In Linux system, you can use the file /etc/resolv.conf  Find nameserver in.
But I don't know how to find the relevant configuration in the window system

**Comment by MEschenbacher** [comment]
Maybe I can put something together from https://docs.microsoft.com/en-us/windows/win32/api/iphlpapi/nf-iphlpapi-getadaptersaddresses

**Comment by lixiangzhong** [comment]
```go
package main

import (
	"fmt"
	"log"
	"os"
	"syscall"
	"unsafe"

	"golang.org/x/sys/windows"
)

func main() {
	var b []byte
	l := uint32(15000) // recommended initial size
	for {
		b = make([]byte, l)
		err := windows.GetAdaptersAddresses(syscall.AF_UNSPEC, windows.GAA_FLAG_INCLUDE_PREFIX, 0, (*windows.IpAdapterAddresses)(unsafe.Pointer(&b[0])), &l)
		if err == nil {
			if l == 0 {
				log.Fatal("nil,nil")
			}
			break
		}
		if err.(syscall.Errno) != syscall.ERROR_BUFFER_OVERFLOW {
			log.Fatal(os.NewSyscallError("getadaptersaddresses", err))
		}
		if l <= uint32(len(b)) {
			log.Fatal(os.NewSyscallError("getadaptersaddresses", err))
		}
	}
	var aas []*windows.IpAdapterAddresses
	for aa := (*windows.IpAdapterAddresses)(unsafe.Pointer(&b[0])); aa != nil; aa = aa.Next {
		aas = append(aas, aa)
	}
	for _, v := range aas {
		fmt.Printf("%+v\n", v.FirstDnsServerAddress.Address.IP().String())
	}
}

```

Maybe this way. I'll find a time to maintain this project

**Comment by lixiangzhong** [comment]
It is implemented in release v1.4.0

**Comment by MEschenbacher** [comment]
Thank you!
